### PR TITLE
USHIFT-323: use remote kubeconfig instead of ssh

### PR DESCRIPTION
**Pull request by pacevedom**

### CODE REVIEW

**Comment by pacevedom** [comment]
/hold
Need to rework binary paths

**Comment by pacevedom** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-e2e-openshift-conformance-sig-network

**Comment by pacevedom** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-e2e-openshift-conformance-sig-api-machinery

**Comment by pacevedom** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-e2e-openshift-conformance-sig-api-machinery

File: ci-operator/step-registry/openshift/microshift/e2e/pre-rpm-install/openshift-microshift-e2e-pre-rpm-install-commands.sh [hunk 1]
```diff
@@ -52,14 +52,26 @@ firewall-cmd --zone=trusted --add-source=10.42.0.0/16 --permanent
...
 chmod +x usr/bin/pre_rpm_install.sh
+
+BASE_DOMAIN="$(cat ${CLUSTER_PROFILE_DIR}/public_hosted_zone)"
+
+mkdir -p "${PAYLOAD_PATH}"/etc/microshift
+cat << EOF > "${PAYLOAD_PATH}"/etc/microshift/config.yaml
+---
+apiServer:
+  subjectAltNames:
+  - ${INSTANCE_PREFIX}.${BASE_DOMAIN}
```
Comment (mangelajo) [inline ci-operator/step-registry/openshift/microshift/e2e/pre-rpm-install/openshift-microshift-e2e-pre-rpm-install-commands.sh:67]: cool, great example! :D

**Pull request merged**